earth_enterprise/src/installer/common.sh

```diff
@@ -250,6 +250,10 @@ create_links()
         ln -s $BASEINSTALLDIR_VAR/run $BASEINSTALLDIR_OPT/run
     fi
 
+    if [ ! -L "$BASEINSTALLDIR_OPT/share/doc" ]; then
```

Comment:
please review , its tested and the check is corrected to verify "if the link does not exists, then"